.golangci.yml

```diff
@@ -118,8 +118,6 @@ linters-settings:
   depguard:
     list-type: blacklist
     include-go-root: false
-    packages:
-      - github.com/sirupsen/logrus

```

Comment:
fix lint error: 
error: `depguard  `github.com/sirupsen/logrus` is in the blacklist`